rust/perspective-viewer/src/themes/vaporwave.less

```diff
@@ -93,45 +94,51 @@ perspective-string-column-style[theme="Vaporwave"] {
     --d3fc-series-9: hsl(157, 100%, 25%);
     --d3fc-series-10: hsl(272, 100%, 35%);
 
-    --d3fc-negative--gradient: linear-gradient(#f3d431,
-            #efb92d,
-            #ed9c25,
-            #eb7e20,
-            #e75d1e,
-            #d14632,
-            #b03e38,
-            #8c3a36,
-            #643633,
-            #07081d) !important;
+    --d3fc-negative--gradient: linear-gradient(
+        #f3d431,
+        #efb92d,
+        #ed9c25,
+        #eb7e20,
+        #e75d1e,
```

Comment:
there were several formatting-only changes like this. I'm not sure how this happened - I have format with save enabled, in VSCode but it should be using the same prettier settings that the linter uses. And the linter accepts it in either style